chain/x/zoracle/internal/keeper/raw_request.go

```diff
@@ -22,6 +22,7 @@ func (k Keeper) GetRawDataRequest(
 ) (types.RawDataRequest, sdk.Error) {
 	store := ctx.KVStore(k.storeKey)
 	if !k.CheckRawDataRequestExists(ctx, requestID, externalID) {
+		// TODO: fix error
```

Comment:
Will we not re-consider this error?

Reply:
It will on a different ticket (duplication error)